securedrop/i18n_tool.py

```diff
@@ -201,15 +202,15 @@ def require_git_email_name(git_dir):
                'git -C {d} config --get user.email > /dev/null'.format(
                    d=git_dir))
         if subprocess.call(cmd, shell=True):  # nosec
-            if six.u('docker') in io.open('/proc/1/cgroup').read():
```

Comment:
Any particular reason to remove the usage of `six`?

Reply:
Yes. This PR includes changes from https://github.com/freedomofpress/securedrop/pull/4482 where that change is explained. Sorry for not documenting it here too:

> Use of six.u was corrupting translator names. According to
https://pythonhosted.org/six/index.html#six.u -- "On Python 2, u()
doesn't know what the encoding of the literal is. Each byte is
converted directly to the unicode codepoint of the same value. Because
of this, it's only safe to use u() with strings of ASCII data."

> Of course some translators' names were not just ASCII, so I've
replaced usage of six.u with u"", as it should be safe on both Python
2 and Python 3 > 3.3, per PEP-0414.